packages/astro/package.json

```diff
@@ -196,13 +195,13 @@
     "remark-code-titles": "^0.1.2",
     "rollup": "^3.9.0",
     "sass": "^1.52.2",
-    "sharp": "^0.31.3",
+    "sharp": "^0.32.1",
     "srcset-parse": "^1.1.0",
     "undici": "^5.22.0",
     "unified": "^10.1.2"
   },
   "peerDependencies": {
-    "sharp": "^0.31.3"
+    "sharp": ">=0.32.0"
```

Comment:
Isn't bumping the peer dep version a breaking change? Unless we keep it `>=0.31.3`

Reply:
Sharp has a bit of a weird relationship with versions, it doesn't really follow semver nor does it changes its API (and either way, is very unlikely to affect us since we only use the very very basic options, users don't have direct access to sharp)

Reply:
The issue is that when the user installs the new version of Astro, the package manager would then warn about the version mismatch, even though it will still work. IIRC some package managers would error out entirely too. So I think we could keep the version to avoid that.

Reply:
ah, yeah that's annoying. Will revert to >=0.31.0 then